remove-node.yml

```diff
@@ -32,8 +32,7 @@
   gather_facts: no
   environment: "{{ proxy_disable_env }}"
   roles:
-    - { role: kubespray-defaults, when: reset_nodes|default(True)|bool }
-    - { role: bootstrap-os, tags: bootstrap-os, when: reset_nodes|default(True)|bool }
+    - { role: kubespray-defaults }
```

Comment:
This change is duplicated with https://github.com/kubernetes-sigs/kubespray/pull/7610
Please check https://github.com/kubernetes-sigs/kubespray/pull/7610#discussion_r634593075